### PR TITLE
Recurring Payments: a little onboarding + atomic fix

### PR DESCRIPTION
**Pull request by artpi**
I realized that 2 groups may stumble upon the new earn tab:
- a) users without context
- b) atomic users

This PR disables feature for Jetpack older than 7.4 (and 7.4 is enabling it). Additionally it shows a little onboarding to explain what is it about.

### Screenshots

Zrzut ekranu 2019-06-1 o 18 58 05
Zrzut ekranu 2019-06-1 o 19 12 17
Zrzut ekranu 2019-06-1 o 19 11 54

### Testing instructions

1. Open in atomic. You should get "Upgrade to jetpack 7.4" Not ideal, but works
2. Open in a site you did not connect. You should get little onboarding.

### CODE REVIEW

File: client/my-sites/earn/memberships/index.jsx [hunk 1]
```diff
@@ -270,12 +270,31 @@ class MembershipsSection extends Component {
 						) }
 					/>
 				) }
+				<SectionHeader label={ this.props.translate( 'About Recurring Payments' ) } />
```
Comment (a8ci18n) [inline client/my-sites/earn/memberships/index.jsx]: Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Recurring Payments' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (artpi) [reply client/my-sites/earn/memberships/index.jsx]: 👎

File: client/my-sites/earn/memberships/index.jsx [hunk 2]
```diff
@@ -259,6 +259,32 @@ class MembershipsSection extends Component {
...
+					<div className="memberships__module-content module-content">
+						<p>
+							{ this.props.translate(
+								'Start collecting subscription payments! Recurring Payments is a feature inside the block editor. When editing a post or a page you can insert a button that will allow you to collect paying subscribers.'
+							) }
+						</p>
+					</div>
+				</Card>
+				<Notice
+					text={ this.props.translate( 'Read more about Recurring Payments feature' ) }
```
Comment (rralian) [inline client/my-sites/earn/memberships/index.jsx]: should be "Read more about the Recurring Payments feature."

File: client/my-sites/earn/memberships/index.jsx [hunk 3]
```diff
@@ -290,30 +317,36 @@ class MembershipsSection extends Component {
...
-						href={ `https://wordpress.com/plugins/jetpack/${ this.props.siteSlug }` }
-						icon="external"
-					/>
-				</Notice>
+				<div>
+					{ this.renderOnboarding() }
+					<Notice
+						status="is-warning"
+						text={ this.props.translate(
+							'Please update Jetpack plugin to version 7.4 or higher in order to use the Recurring Payments button block'
```
Comment (rralian) [inline client/my-sites/earn/memberships/index.jsx]: "Please update the Jetpack plugin [...]"

Comment (artpi) [reply client/my-sites/earn/memberships/index.jsx]: Fixed!

File: client/my-sites/earn/memberships/index.jsx (hunk 2 above)
Comment (artpi) [reply client/my-sites/earn/memberships/index.jsx]: fixed!

**Pull request merged**